docs/en/platform-engineers/cloud-services.md

```diff
@@ -2,15 +2,16 @@
 
 In this tutorial, we will add a Alibaba Cloud's RDS service as a new workload type in KubeVela.
 
-## Step 1: Install and configure Crossplane 
+## Step 1: Install and configure Crossplane
 
-In this tutorial, we use Crossplane as the cloud resource operator for Kubernetes.   
-Please follow the Crossplane [Documentation](https://crossplane.io/docs/) under the `Welcome` section to configure Crossplane with your cloud account.   
-**Note: When installing crossplane helm chart, please do not include the `--set alpha.oam.enabled=true` flag as KubeVela has already installed these OAM crds.**
+We use Crossplane as the cloud resource operator for Kubernetes. This tutorial has been verified with Crossplane `version 0.14`.
+Please follow the Crossplane [Documentation](https://crossplane.io/docs/), especially the `Install & Configure` and `Compose Infrastructure` sections to configure Crossplane with your cloud account.
+
+**Note: When installing crossplane helm chart, please don't set `alpha.oam.enabled=true` as OAM crds are already installed by KubeVela.**
 
 ## Step 2: Add Workload Definition
 
-First,register the `rds` workload type to KubeVela:
+First, register the `rds` workload type to KubeVela.
 
 ```bash
 $ cat << EOF | kubectl apply -f -
```

Comment:
This is cool which provides an example for applying an workload/traits besides built-in and `vela cap`.